.github/scripts/trymerge.py

```diff
@@ -1292,7 +1300,7 @@ def post_comment(msg: str) -> None:
     repo.push(pr.default_branch(), dry_run)
     post_comment(f"@{pr.get_pr_creator_login()} your PR has been successfully reverted.")
     if not dry_run:
-        gh_add_labels(pr.org, pr.project, pr.pr_num, ["reverted"])
+        pr.add_numbered_label("reverted")
```

Comment:
What do you think about removing the reverted label during merges instead of adding numbered versions of it a new revert happens?

So the PR would only have a reverted label if it's currently reverted.

Reply:
@kit1980 was against this idea, as he relies on reverted label to know if PR was ever reverted

Reply:
No, actually that would be fine for my purposes of counting contributions. If it's reverted and then re-merged again, I don't need to know if it was reverted at some point. But for other purposes it may be needed.

Also with this PR we'll have additional info like some PRs were reverted at least 3 times, and we can use it to find some especially problematic use cases.